frontend/public/locales/en/apply-child.json

```diff
@@ -512,20 +512,38 @@
     "email-address": "Email address: {{email}}",
     "lang-pref-title": "Language preference",
     "lang-pref-change": "Change language preference",
-    "dental-title": "Access to other dental insurance or coverage",
-    "dental-insurance-title": "Access to dental insurance",
-    "dental-insurance-change": "Change answer to dental insurance",
-    "dental-benefit-title": "Access to government dental benefit",
-    "dental-benefit-change": "Change answer to public dental benefits",
-    "dental-benefit-has-access": "Has access to the following benefits:",
-    "dental-benefit-has-no-access": "Has access to the following benefits:",
     "submit-app-title": "Submit your application",
     "submit-p-proceed": "By proceeding with this application, you are agreeing to attest truthfully to the information provided.",
     "submit-p-false-info": "If you provide false information on your application, you and others you applied for may be removed from the plan.",
     "submit-p-repayment": "If you or your family members were not eligible to apply, you or your family member may have to repay the full cost of care received through the Canadian Dental Care Plan.",
+    "continue-button": "Continue",
+    "back-button": "Back",
     "submit-button": "Submit Application",
     "exit-button": "Exit application",
+    "yes": "Yes",
+    "no": "No"
+  },
+  "review-child-information": {
+    "read-carefully": "Please review this information carefully. The information provided will be used by Sun Life to create your child(ren)'s member account.",
+    "page-title": "Review child(ren) information",
+    "page-sub-title": "{{child}} information",
+    "dob-title": "Date of birth",
+    "dob-change": "Change date of birth",
+    "is-parent": "Parent or legal guardian of child",
+    "sin-title": "Social Insurance Number (SIN)",
+    "sin-change": "Change SIN",
+    "full-name-title": "Full name",
+    "full-name-change": "Change full name",
+    "dental-title": "{{child}} access to dental insurance",
+    "dental-insurance-title": "Access to other dental insurance or coverage",
+    "dental-insurance-change": "Change answer to dental insurance",
+    "dental-benefit-title": "Access to government dental benefit",
+    "dental-benefit-change": "Change answer to public dental benefits",
+    "dental-benefit-has-access": "Has access to the following benefits:",
+    "dental-benefit-has-no-access": "Has access to the following benefits:",
+    "continue-button": "Continue",
     "back-button": "Back",
+    "exit-button": "Exit application",
```

Comment:
That key shouldn't exist in the "child" flow context for "review-child-information".

Reply:
Removed unused translation